inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/management/DynamicWorkloadManagementPage.html

```diff
@@ -17,84 +17,112 @@
  - limitations under the License.
 -->
 <html xmlns:wicket="http://wicket.apache.org" xmlns="http://www.w3.org/1999/html" lang="">
-<head>
 <wicket:head>
-  <style type="text/css">
-.page-content {
- overflow: auto;
-}
+    <style type="text/css">
+        .page-content {
+            overflow: auto;
+        }
 
-.table-striped>tbody>tr:nth-child(2n+1)>td, .table-striped>tbody>tr:nth-child(2n+1)>th
- {
- background-color: lightgray;
-}
+        .table-striped > tbody > tr:nth-child(2n+1) > td, .table-striped > tbody > tr:nth-child(2n+1) > th {
+            background-color: lightgray;
+        }
 
-.table-hover tbody tr:hover td, .table-hover tbody tr:hover th {
- background-color: burlywood;
-}
-</style>
+        .table-hover tbody tr:hover td, .table-hover tbody tr:hover th {
```

Comment:
Let's stick to standard colors defined by the overall theme. Why override the color here?

Reply:
At least on my screen:
I can not see the difference in the default color theme. The difference is that minor, that it all simply looks the same (nearly white). 
Same for the hovering. 
I found the colors I picked quite good to work with.
However, ofc I can simply remove the override to stick to the default theme.

Reply:
The choice of color is one of taste. But the issue of overwriting colors or other styles locally is one of maintainability and consistency. The more we override style locally, the more effort we have when upgrading to e.g. a newer version of Bootstrap (I felt that because I did that earlier this year...).